sycl/test-e2e/Matrix/joint_matrix_hip_gfx90a.cpp

```diff
@@ -6,10 +6,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-// RUN: %{build} -fsycl -fsycl-targets=amd_gpu_gfx90a %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=amd_gpu_gfx90a %s -o %t.out
```

Comment:
This change is a bug fix, we shouldn't use `%{build}` if we're setting `-fsycl-targets`/choosing file and output